### ISSUE TITLE
How to provide suggestions/corrections?

### ISSUE THREAD

**Issue by eddieantonio** [kpatenio/WSL-Documentation#1]
Katherine! Happy linuxing!

If I want to provide suggestions and/or corrections to the existing material, how's best to do that? A simple pull request? Would you like me to get in contact with you in some other way beforehand?

Ubuntu package management can get confusing, and there's a lot of package managers installing package managers and installing packages by piping a shell script into a sudo bash shell <kbd>#yikes</kbd>

**Comment by kpatenio** [comment]
Hi @eddieantonio! I'll admit I never really thought of it! 😅 I really appreciate you reaching out and wanting to help! Like seriously, it really means a lot to me. I thought I understood Linux (and the Ubuntu distro), but turns out I don't! There's so much more to Linux than I thought 🙃.

For the time being, I guess PRs are a good way to start. Besides, I don't always get the opportunity to review/accept/merge PRs. 😉  Another way we could do this is to create issues! I never really had anyone contribute to one of my own repos before, so if you have any other suggestions or ideas, let me know!

**Comment by kpatenio** [comment]
Admittedly, all the notes I've made so far, I made all in one night. I played around with WSL without any intention of making this repository. However, over time, I faced information overload hehe, and I needed to keep track of everything I was learning. 

I will be revising my notes often. If you have any corrections and/or suggestions, do let me know! I'd rather be corrected than be learning the wrong things!
